### PR TITLE
chore: drop dependency on common

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -364,10 +363,6 @@ function Datastore(options) {
     return new Datastore(options);
   }
 
-  options = common.util.normalizeArguments(this, options, {
```
Comment (ofrobots) [inline src/index.js]: Normalization of arguments is no longer necessary?

File: test/request.js [hunk 2]
```diff
@@ -23,24 +23,31 @@ var is = require('is');
 var proxyquire = require('proxyquire');
 var sinon = require('sinon').sandbox.create();
 var through = require('through2');
-var util = require('@google-cloud/common').util;
+var pfy = require('@google-cloud/promisify');
```
Comment (ofrobots) [inline test/request.js]: style nit: cute but unnecessary abbreviation.

File: src/index.js (hunk 1 above)
Comment (JustinBeckwith) [reply src/index.js]: I believe this was an artifact of the big `google-cloud` npm module.  @stephenplusplus to verify :)

Comment (stephenplusplus) [reply src/index.js]: Yep, that's right. All this did was extend the global configuration:

```js
var gcloud = require('gcloud')({ projectId: 'a' })

var datastore = gcloud.datastore()
// datastore.projectId = 'a'

var gcs = gcloud.storage({ projectId: 'b' })
// gcs.projectId = 'b'
```

**Pull request merged**